src/UXClient/Components/LinePlot/LinePlot.ts

```diff
@@ -131,6 +133,25 @@ class LinePlot extends Plot {
                 .ticks(Math.max(2, Math.ceil(this.height/(this.yAxisState.axisType === YAxisStates.Stacked ? this.visibleAggCount : 1)/90))))
                 .selectAll("text").classed("standardYAxisText", true)
         }
+
+        // If yAxisOnClick present, attach to yAxis
+        if(yAxisHasOnClick){
+            yAxis.on("click", () => {
+                yAxisOnClick();
+            })
+            let label = document.getElementsByClassName(`tsi-swimLaneLabel-${agg.swimLane}`)[0];
+            if(label){
+                yAxis.on("mouseover", () => {
```

Comment:
is this part dependent on the axis's having onclick? I think it does not have to right? Like even though there is no onclick function in options, on hover it would be nice to see the css class changes? Maybe onclick is different here than swimlaneoptions' onclick.

other thing here, can't we do it with css styling rather than adding mouseover/out programmatically here?

Reply:
This needs a programmatic `mouseover` because the hover action on the y-axis needs to also trigger a bold font-weight on the y-axis label.

I think if the onClick isn't set, or isn't a function, the axis / label shouldn't have the bold hover effect.  That way the user won't try to click them if there is no action.

Reply:
You can hover and click both the y-axis and the y-axis label.  These both are different D3 elements that live in different files.  The logic makes sure that hovering a label (with an onClick present) will bold both that label and its associated axis, and vice-versa.